dev/dns_test.go

```diff
@@ -44,6 +44,7 @@ func TestServeDNS(t *testing.T) {
 
 				return nil
 			},
+			retry.Attempts(3),
```

Comment:
limit retries to 3 in tests to help fail faster. added while debugging another issue.